app/brave_settings_strings.grdp

```diff
@@ -548,13 +548,16 @@
     Google Sign-In
   </message>
   <message name="IDS_SETTINGS_SITE_SETTINGS_GOOGLE_SIGN_IN_ASK" desc="Label for G sign-in site settings.">
-    Ask when a site wants to use legacy Google Sign-In
+    Sites can ask to use legacy Google Sign-In via third-party cookies
   </message>
   <message name="IDS_SETTINGS_SITE_SETTINGS_GOOGLE_SIGN_IN_BLOCK" desc="Label for G sign-in site settings.">
-    Block sites from using legacy Google Sign-In
+    Don't allow legacy Google Sign-In via third-party cookies
   </message>
```

Comment:
Can this message be removed now that it's not being used?

Reply:
We use it in `browser/resources/settings/brave_overrides/site_settings_page.ts` for the label to be displayed 
<img width="943" alt="image" src="https://user-images.githubusercontent.com/5284154/217451347-617d526d-fc5e-439d-aad2-0d7246e7957f.png">

Reply:
Seems like it's still being used in `site_settings_page` - that intentional?

Reply:
Yes, I believe we need some text indicating what the current default value is (ask or block) in `site_settings_page`.